lib/js/patterns/dropdown.js

```diff
@@ -0,0 +1,54 @@
+// Cool Helpers
+import * as dom from '../helpers/dom'
+import * as classy from '../helpers/classy'
+import * as aria from '../helpers/aria'
+import * as event from '../helpers/event'
+import bus from '../helpers/bus'
+
+// ┌──────────┐
+// │ Dropdown │
+// └──────────┘
+// show and hide dropdown menus
+function dropdown () {
+  var toggles = dom.findElements('.js-dropdown-toggle');
+  var dropdowns = dom.findElements('.js-dropdown');
+
+  bus.on('dropdown:toggle', toggleDropdown)
+  bus.on('dropdown:close', closeAllDropdowns)
+  bus.on('keyboard:escape', closeAllDropdowns)
+  bus.on('dropdown:bind', bindDropdowns)
+
+  function closeAllDropdowns (options) {
+    event.remove(document.body, event.click(), closeAllDropdowns);
+    dropdowns.forEach(function (dropdown) {
+      classy.remove(dropdown, 'is-active');
+    });
+  }
+
+  function toggleDropdown (options) {
+    if (!options) return
+    classy.toggle(options.node, 'is-active');
+    if(classy.has(options.node, 'is-active')) {
+      event.add(document.body, event.click(), closeAllDropdowns);
+    }
+  }
+
+  function bindDropdowns (options) {
+    if (!options) {
```

Comment:
no, good catch. Was going to allow for binding a single node. Never wrote that logic, didnt extend it out